drizzlepac/alignimages.py

```diff
@@ -450,39 +450,47 @@ def run_align(input_list, archive=False, clobber=False, debug=False, update_hdr_
         # reset process
         temp_imglist = []
         fit_info_dict = OrderedDict()
-        for catalogIndex in range(0, len(
-                catalog_list)):  # loop over astrometric catalog
-            log.info("{} STEP 5: Detect astrometric sources {}".format("-" * 20, "-" * 48))
-            log.info("Astrometric Catalog: %s", str(catalog_list[catalogIndex]))
-            reference_catalog = generate_astrometric_catalog(process_list,
-                                                             catalog=catalog_list[catalogIndex],
-                                                             output=output)
+        reference_catalog_dict={}
+        for algorithm_name in fit_algorithm_list:  # loop over fit algorithm type
+            for catalogIndex in range(0, len(catalog_list)):  # loop over astrometric catalog
```

Comment:
```suggestion
            for catalogIndex, catalog_name in enumerate(catalog_list):  # loop over astrometric catalog
```
This would eliminate the need for line 457. Where the only entries in `catalog_list` are strings, there is no need to cast them to strings in line 457 either.

Reply:
Resolved issue with commit [bf5ab62](https://github.com/spacetelescope/drizzlepac/pull/328/commits/bf5ab623c7cd17cc5e11cbdf9572828829fa5235)